src/test/java/teammates/test/driver/BackDoor.java

```diff
@@ -555,6 +555,13 @@ public static String deleteFeedbackQuestion(String questionId) {
     private void ____FEEDBACK_RESPONSE_level_methods______________________________() {
     }
     
+    public static String createFeedbackResponse(FeedbackResponseAttributes feedbackResponse) {
+        DataBundle dataBundle = new DataBundle();
+        dataBundle.feedbackResponses.put("dummy-key", feedbackResponse);
+        return persistNewDataBundle(Utils.getTeammatesGson()
+                .toJson(dataBundle));
```

Comment:
No need for the line wrapping at all? 
8 space indentation is the last resort when line wrapping. There are better alternatives. It's ok to use the best indentation, even if other methods don't use it.